aiosmtpd/smtp.py

```diff
@@ -160,16 +155,14 @@ def eof_received(self):
     def _set_post_data_state(self):
         """Reset state variables to their post-DATA state."""
         self.envelope = self._create_envelope()
-        self.require_SMTPUTF8 = False
 
     def _set_rset_state(self):
         """Reset all state variables except the greeting."""
         self._set_post_data_state()
 
     @asyncio.coroutine
     def push(self, status):
-        response = bytes(
-            status + '\r\n', 'utf-8' if self.require_SMTPUTF8 else 'ascii')
+        response = bytes(status + '\r\n', encoding='ascii')
```

Comment:
I think this will break non-ASCII [responses to VRFY](https://tools.ietf.org/html/rfc6531#section-3.7.4.2) (and EXPN but we don't yet support that).